### DISCUSSION TITLE
OBS Studio WebSocket and M3

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2573] [General]
The option **obs-websocket v4**, is no longer present in OBS Studio, and I have seen some problem in the integration with OBS Studio, do not know how, but this option ends up being activated and loses communication with OBS Studio (WebSocket Server). **Is it possible to remove this option (_Use obs-websocket v4_) from M3 and leave only the first option of WebSocket ?**

![image](https://github.com/user-attachments/assets/26da8a98-9e44-4d3e-b9c3-a45ccf3f03bf)

**Comment by sircharlo** [comment]
obs-websocket v4 has indeed been deprecated in favor of v5; this option has therefore been removed from the new version of the app.
